CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@
 - [cygnus-ngsi][hardening] Pre-aggregate batches in NGSISTHSink (#1359)
```

Comment:
Yes. At most, something could be commented somewhere in the doc regarding the possibility of using file channels instead of memory channels. The user who warned about the issue said nothing was said about this in the doc... Strictly, this is not necessary since Cygnus inherits from Flume, which allows file channels, thus it is understood Cygnus allows such type of channel as well. But could be fine to say something.